### DISCUSSION TITLE
Custom Board Shapes

### DISCUSSION THREAD

**Discussion by naomi-lgbt** [BattlesnakeOfficial/feedback#56] [Feature Requests]
Came up on the stream, ideas to allow custom board shapes.

Could be as simple as a rectangle, or as advanced as allowing players to create boards from shapes (like a star!).

**Comment by coreyja** [comment]
I think one way to potentially implement this is via walls on a 'regular' rectangle board.

Inside the normal board, if you could have `walls` you could build all kinda of different shapes inside the 'standard' rectangles!
Might make implementing custom shapes easier, and could be used in other game modes (like the Pacman game mode suggestion)
